protocol/src/protocol.ts

```diff
@@ -1046,7 +1046,7 @@ export namespace Project {
 	export const descriptor = new VertexDescriptor<Project>(Object.assign({}, V.descriptor.description, {
 		label: VertexLabels.property(VertexLabels.project),
 		kind: new StringProperty(),
-		name: new StringProperty(),
+		name: new StringProperty(PropertyFlags.optional),

```

Comment:
This is from the [other open PR](https://github.com/microsoft/lsif-node/pull/145); however, it's required for the added C# LSIF to pass validation